### PR TITLE
Don't autoload report, removed required option

### PR DESCRIPTION
**Pull request by kkrampa**
@czue @NoahCarnahan 
Since required option allowed to load report with blank filters (it just checked if proper params were send in request) I think we can drop required option. At the initial load user is informed that he needs to click apply button.

### CODE REVIEW

File: corehq/apps/reports_core/templates/reports_core/base_template_new.html [hunk 1]
```diff
@@ -181,20 +183,27 @@
...
-    <table id="report_table_{{ report.slug }}" class="table table-striped datatable">
-        <thead>
-        {{ headers.render_html|safe }}
-        </thead>
-        <tbody>
-        </tbody>
-    </table>
+<div id="reportHint" class="well">
+    <h4>Why can't I see any data?</h4>
+    <p>Please choose your filters above and click <strong>Apply</strong> to see report data.</p>
```
Comment (czue) [inline corehq/apps/reports_core/templates/reports_core/base_template_new.html]: should have translation markup

**Comment by czue** [comment]
@NoahCarnahan do you remember why we added required? I don't recall offhand.

**Comment by czue** [comment]
(didn't review closely - will let @NoahCarnahan take a look at this)

**Comment by NoahCarnahan** [comment]
@czue: I don't know what use case we were imagining when we added "required"

@kkrampa: PR looks good. Wrap the warning message in `{% blocktans %} ... {% endblocktrans %}` tags and it should be good to go. Thanks!

**Comment by kkrampa** [comment]
@NoahCarnahan Updated

**Comment by NoahCarnahan** [comment]
Thanks!

**Pull request merged**